understanding/22/focus-not-obscured-minimum.html

```diff
@@ -36,13 +36,18 @@ <h2>Intent of Focus Not Obscured</h2>
  
          <p>Another form of obscuring can occur due to light boxes or other semi-opaque effects overlapping the item with focus. While less than 100 percent opacity is not causing the component to be "entirely hidden," such semi-opaque overlaps may cause a failure of <a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html">2.4.11 Focus Appearance</a>. When a focus indicator can be covered by a semi-opaque component, the ability of the focus indicator to pass 2.4.11 should be evaluated (and pass) while the focus indicator is under the semi-opaque component. The intention in both situations is that the component receiving focus should never be obscured to the point a user cannot tell which item has focus.</p>
 
-      <p class="note">If the interface is configurable so that the user can move toolbars and non-modal dialogs around, then only the initial positions of user-movable content would be considered for testing and conformance of this Success Criterion.</p>
+        <p>Where other content can overlap with a focused item, the focused element should not be hidden.  Typical types of content that can overlap focused items are sticky footers, sticky headers, or non-modal dialogues.  As a user tabs through the page (thus moving the keyboard focus indicator), these layers of content can obscure the focused item, including the focus indicator.  Such behavior is usually indiccative of a failure to satisfy this success criterion.</p>
 
+        <p>If the interface is configurable so that the user can manually open, close, and/or move components such as toolbars and non-modal dialogues, then only the initial default position and state of user-movable content is applicable for testing and conformance of this success criterion.  Similarly, if the interface contains content where user interation causes eleements to open/close or expand/collapse (i.e., a change is initiated by the user), only the initial default state of user-controlable components is in scope for this success criterion.</p>
+
+        <p>Converserely, if a component automatically opens and obscures focused elements, that behavior would fail this criterion.  For example, if a navigation drop-down opens upon focus, but does not close when focus moves to other element, the page would not pass if focusable elements were visually obscured.</p>
```

Comment:
IMO, a dropdown that opens on focus is verging on a failure of On Focus (not enough context to know) and is not a behaviour we should be advancing. So I don't think this serves as a good example.